python-api/tests/unittests/test_connection_unit.py

```diff
@@ -147,9 +147,65 @@ def test_mock2(self, monkeypatch):
         assert conn.connection is not None
 
 
+class TestSignalSubscribe(object):
+
+    """Unittests for signal_subscribe"""
+
+    def test_handler_not_callable(self, monkeypatch):
+        """Test if handler is not callable"""
+
+        conn = Connection()
+        test_cbs = ['', None, [], {}]
+        for test_cb in test_cbs:
+            with pytest.raises(ValueError):
+                conn.signal_subscribe(test_cb)
+
+    # pylint: disable=unpacking-non-sequence
```

Comment:
Can you add a comment on why this is disabled. I assume it's because of the `args, _ = signal_subscribe_mock.call_args` line?

BTW You can disable it just for one line too.